keras_cv/layers/preprocessing/random_choice_test.py

```diff
@@ -53,6 +53,14 @@ def call_pipeline(xs):
 
         self.assertAllClose(xs + 1, os)
 
+    def test_batchwise(self):
+        layer = AddOneToInputs()
+        pipeline = layers.RandomChoice(layers=[layer], batchwise=True)
+        xs = tf.random.uniform((4, 5, 5, 3), 0, 100, dtype=tf.float32)
+        os = pipeline(xs)
+
+        self.assertAllClose(xs + 1, os)
```

Comment:
Does this meaningfully test the batchwise behavior? It seems like it doesn't but maybe I'm missing somethiing

Reply:
Good point; I updated it to reference `call_counter` in the `AddOne` layer to ensure we are testing this behavior.